public/index.html

```diff
@@ -1,14 +1,12 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+
+<head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/NERfavicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
+    <meta name="description" content="Web site created using create-react-app" />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```

Comment:
what is this logo?

Reply:
logo192.png is the previous logo used in the previous version. Its the logo that looks like an atom.

Reply:
Oh wait I should update all instances of the default react logo with the favicon right?
Sorry about that, thought it was needed in the one place.

Reply:
No worries at all! Yeah, scrubbing out all instances of the default React logo would be perfect. Thank you!